Automattic/wp-e2e-tests-for-branches
# Using webhook script instead of CircleCI

This will use a script to return build results to GitHub instead of the unreliable CircleCI notify functionality

To Test:

- Run wp-e2e-tests-gh-bridge locally after making sure that `E2E_WRAPPER_BRANCH` environment variable is set to `try/use-webhook`
- Post a PR payload to the local instance
- Confirm that after the tests complete, the results show up on the PR in GitHub


part of https://github.com/Automattic/wp-e2e-tests/issues/1668
